queries/ocaml/hightlights.scm

```diff
@@ -0,0 +1,146 @@
+(module_name) @structure
```

Comment:
Structure? I dont think we have this highlight (too lazy to check on mobile). Maybe `@namespace`?

Reply:
We do have a structure highlight `TSStructure`

Reply:
I could use either, but structure works.